app/scripts/modules/core/src/instance/details/InstanceDetails.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+import { Overridable, IOverridableProps } from 'core/overrideRegistry';
+import { ReactInjector } from 'core/reactShims';
+import { Application } from 'core/application';
+import { Spinner } from 'core/widgets';
+
+export interface IInstanceDetailsProps extends IOverridableProps {
+  $stateParams: {
+    provider: string;
+    instanceId: string;
+  },
+  app: Application;
+}
+
+export class InstanceDetails extends React.Component<IInstanceDetailsProps, { accountId: string }> {
```

Comment:
This component fetches the account name for an instance from the `VersionedCloudProviderService` .
It then uses the `accountId`  to render the `@Overridable` component.